clang-tools-extra/include-cleaner/test/tool.cpp

```diff
@@ -49,12 +49,15 @@ int x = foo();
 //        RUN: FileCheck --match-full-lines --check-prefix=EDIT2 %s < %t.cpp
 //  EDIT2-NOT: {{^}}#include "foo.h"{{$}}
 
+// This test has commands that rely on shell capabilities that won't execute
+// correctly on Windows e.g. subshell execution
+//   REQUIRES: shell
```

Comment:
this doesn't just affect that particular run, affects the whole, can you move the test into its own file ?

Reply:
This was explained in the previous comment. It now works on Windows as well.